mmv1/third_party/terraform/.teamcity/components/inputs/services_beta.kt

```diff
@@ -496,6 +496,11 @@ var ServicesListBeta = mapOf(
         "displayName" to "Memcache",
         "path" to "./google-beta/services/memcache"
     ),
+    "memorystore" to mapOf(
```

Comment:
I see the failure: https://github.com/GoogleCloudPlatform/magic-modules/actions/runs/10532606319/job/29187084684?pr=11463
This is not a GA service yet. Do I need to add it to mmv1/third_party/terraform/.teamcity/components/inputs/services_ga.kt ?

Reply:
I don't think you need to. I believe these are used for our nightly tests of the corresponding provider.